### DISCUSSION TITLE
Supabase Google Auth and Magic Links not working on Mobile

### DISCUSSION THREAD

**Discussion by fdarkaou** [supabase/supabase#13968] [Questions]
Hi all,

I have a VueJS frontend using Supabase for authentication. Users can either log in using Google Auth or through a Magic Link - on desktop all of this works perfectly. On mobile however, neither option seems to work. Users are redirected to the landing page and are not being logged in.

What could be the reason for this? When I simulate a mobile device using Google DevTools it works fine as well, only on an actual mobile device it does not work.

Would really appreciate your input!!

**Comment by Chippd** [comment]
Dropping a comment here to say I'm having the same issue, as mentioned in Discord:
>  I have a Nuxt app and my users are not being logged in when clicking the link. 
As far as I can tell, it's almost like the cookie isn't being dropped in time when there's any kind of latency, like the page at https://[appid].supabase.co/auth/v1/verify is redirecting before the cookie is dropped, and therefore our apps don't detect the session. my app is https://aihairstyles.com/ if anyone wants to try it out
Seems most egregious as an issue on mobile ☝️

**Comment by james-william-r** [comment]
...also having this issue. Totally functional from a desktop, but oAuth doesn't work at all when it comes to Safari iOS. Seems to be something to do trouble accessing a web api. This is the error kicked out in dev build:
![IMG_9498](https://github.com/supabase/supabase/assets/66140148/50fdcc54-a27d-4b61-a308-55f8973513b0)

**Reply by james-william-r** [reply]
So, this is actually expected if you're running a dev build locally and don't have an SSL certificate set up. - Also, I'm not having trouble with login on mobile anymore! I removed a router refresh/reload that I'd seen somewhere in a guide and included after login function, think it was an outdated guide though. 🤷

**Comment by fuaditrockz** [comment]
Are supabase resolve this issue? It happen on my project too.
